google-cloud-spanner/src/main/java/com/google/cloud/spanner/SimpleClientImpl.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.spanner;
+
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class SimpleClientImpl implements SimpleClient {
+
+  private final DatabaseClient client;
+  private static final Logger logger = Logger.getLogger(SimpleClientImpl.class.getName());
+
+  SimpleClientImpl(DatabaseClient client) {
+    this.client = client;
+  }
+
+  @Override
+  public ResultSet executeSqlQuery(String query) {
+    return executeSqlQuery(query, TimestampBound.strong());
+  }
+
+  @Override
+  public ResultSet executeSnapshotSqlQuery(String query) {
+    return executeSqlQuery(query, TimestampBound.ofExactStaleness(5L, TimeUnit.SECONDS));
```

Comment:
I don't really see the use case for this method, The staleness is fixed at exactly 5 seconds, which is not a value that has any special meaning. Reading with an exact staleness means that you are reading the data as they were at one exact moment. AFAIK, this request cannot be served more efficiently than reading with `Timestamp.strong()`, so it also doesn't bring any performance gain over the `executeSqlQuery(String)` method.

Reply:
Reading from a snapshot will be more performant. 
I can make the staleness to be 30 sec which should be a more substantial perf improvment.
Let me do that.